server2/models/user.js

```diff
@@ -2,7 +2,13 @@ const { Sequelize, DataTypes, Model } = require('sequelize');
 const dsn = require('../database/db_init');
 const sequelize = new Sequelize(dsn.dbname, dsn.user, dsn.password, {
   host: dsn.host,
-  dialect: 'mysql'
+  dialect: 'mysql',
+  define: {
+    tableName: "users",
+    timestamps: false,
+    createdAt: "create_at",
+    updatedAt: "update_at",
+  },
 });
 
 module.exports = User = sequelize.define('User', {
```

Comment:
Why there is no id for a user? CreatedAt is also missing. And there is no hash in this table, then how you are going to compare the user hash with the password? Why there is no indexing in this table?

Reply:
the user schema is defined below after  module.exports = User = sequelize.define('User', {